src/styles/blocks/card.scss

```diff
@@ -0,0 +1,89 @@
+.card {
+  width: 200px;
+
+  @include flex('', '');
+
+  flex-direction: column;
+  border: 1px solid $card-border-color;
+  border-radius: 5px;
+  background-color: $card-bg-color;
+
+  &__image {
+    margin: 32px 19px 40px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__product-title {
+    margin: 0 16px;
```

Comment:
You should set padding to the card instead, so it would be easier to control.
As for now all of your tests fail, and that's one of the reasons